nyxx_interactions/lib/src/Interactions.dart

```diff
@@ -85,11 +85,13 @@ class Interactions {
         _groupSlashCommandBuilders(commandPartition.last);
 
     final globalBody = globalCommands
-      .where((builder) => builder.permissions.isNotEmpty)
+      .where((builder) => builder.permissions != null)
+      .where((builder) => builder.permissions!.isNotEmpty)
```

Comment:
should be combined together to avoit multiple iteration over colleciton

Reply:
fixed